README.md

```diff
@@ -14,6 +14,10 @@ To run the whole thing, simply run make:
 ```
 make
 ```
+to set agent hostname, required by some tests, use AGENT_HOSTNAME parameter. (Agent expects to have same krb setup as expected by [jck](https://web.archive.org/web/20201126185131/https://icedtea.classpath.org/wiki/JCKDistilled#kerberos_prep). Use your domain for service prinicpal, e.g. http/service.example.com).
+```
+make AGENT_HOSTNAME=agent.example.com
```

Comment:
Waht will happen when the variable is not set?

Reply:
It will fail somewhere unless `SKIP_AGENT_TESTS=1`.  :)

Reply:
The exception then will be very criptic (if any)
Just thoughts - include our include our principla setup script. Provide some default aka "you .have.not.set.your.agent.useagent.hostname.var.or.skip.agent.tests" hmm?

Reply:
I am thinking about creating check in beginning of krb tests, which would fail them, but give some instructions..

Reply:
that would be enough. ty. its ok as additional chnageset.

Reply:
Addressed in another PR: https://github.com/rh-openjdk/CryptoTest/pull/50